### PR TITLE
Portfolio Finish mobile version

### PR DESCRIPTION
**Pull request by Nelofarzabi**
Understand how to parse a Figma design to create a UI.
Use Flexbox to place elements in the page.
Use CSS Grid to place elements in the page.
Build a personal portfolio site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,6 +61,260 @@ <h1 class="name">
...
+        >
+        <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      </div>
+      <hr />
+      <div class="skills-container">
+        <a class="skills--title-link" href="#">
+          <span class="skills--title">Skills</span></a
+        >
+        <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
+      </div>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] You can wrap the three main technologies sections with a `ul` tag as well. Kindly do so. You can use something like the following:

```javascript
 <ul>
    <li>
        <>Languages</>
        <ul>
            <li>Javascript</li>
            <li>...</li>
            <li>...</li>
        </ul>
    </li>
    <li>
        <>Frameworks<>
        <ul>
            <li>...</li>
            <li>...</li>
        </ul>
    </li>
    <li>
        <>Skills</>
        <ul>
            <li>...</li>
            <li>...</li>
        </ul>
    </li>
</ul>
```

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Nelofarzabi, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -115,3 +115,297 @@ li {
 .logo:hover {
   cursor: pointer;
 }
+
+.image-fluid {
+  max-width: 295px;
```
Comment (tufoinnkuo10) [inline style.css:120]: - [optional] please kindly remove most of the max-width you have set and set them to 100% to make your work responsive right up to the desktop version.(Especially the width you gave the whole body (line 23 of style.css file))

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Nelofarzabi 

Your project is complete! Great job  🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- great design
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**